### DISCUSSION TITLE
Renaming `OR` queries to `ANY`

### DISCUSSION THREAD

**Discussion by RobertCraigie** [RobertCraigie/prisma-client-py#408] [Polls]
I've received feedback that suggested using `ANY` instead of `OR` to structure queries. This means that this query:

```py
posts = await client.post.find_many(
    where={
        'OR': [
            {'title': {'contains': 'prisma'}},
            {'content': {'contains': 'prisma'}},
        ]
    }
)
```

Would become:

```py
posts = await client.post.find_many(
    where={
        'ANY': [
            {'title': {'contains': 'prisma'}},
            {'content': {'contains': 'prisma'}},
        ]
    }
)
```

This can make it more clear what the query is actually doing.

It should be noted that this change would have to be completed over a single version bump (`0.6.6` -> `0.7.0`) due to a mypy limitation.

**Comment by TheOnlyWayUp** [comment]
`ANY` imo is more descriptive compared to `OR` :)

**Reply by dv1x3r** [reply]
Original prisma library uses OR. I also think that OR is a good option to keep. And and Or are just standard logic operators in every system, and should not be separated. 

AND 🤝 OR

https://www.prisma.io/docs/concepts/components/prisma-client/filtering-and-sorting
